deployment/script/build-CCNP.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -o errexit
+
+ccnp_url=https://github.com/intel/confidential-cloud-native-primitives.git
+device_plugin=docker.io/library/ccnp-device-plugin:0.1
```

Comment:
What's the usage of this "device_plugin" docker image in the whole CCNP deployment? I only see the docker build  later.